pandas/core/window/rolling.py

```diff
@@ -487,6 +487,36 @@ def _get_window_indexer(self, window: int) -> BaseIndexer:
             return VariableWindowIndexer(index_array=self._on.asi8, window_size=window)
         return FixedWindowIndexer(window_size=window)
 
+    def _apply_blockwise(self, homogeneous_func: Callable) -> FrameOrSeries:
```

Comment:
```suggestion
    def _apply_blockwise(
        self, homogeneous_func: Callable[..., ArrayLike]
    ) -> FrameOrSeries:
```

will get mypy to green. (but not yet checked that the type is correct, just consistent with the variable annotation declaration in the method body)